package.json

```diff
@@ -224,7 +224,6 @@
     "precommit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
     "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
     "rm": "node bin/rm.js",
-    "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
```

Comment:
If we're not using `trim-shrinkwrap`, can we remove it?

Reply:
yup!

Reply:
done

Reply:
We need the `touch node_modules` so `install-if-deps-outdated.js` can keep working.